assets/src/design-system/theme/global/overrideFocusOutline.js

```diff
@@ -34,8 +34,8 @@ export const OverrideFocusOutline = createGlobalStyle`
     This will hide the focus indicator if the element receives focus via the mouse,
     but it will still show up on keyboard focus.
     */
-    .js-focus-visible :focus:not(.${FOCUS_VISIBLE_SELECTOR}),
-    .js-focus-visible :focus-visible:not(.${FOCUS_VISIBLE_SELECTOR}) {
+    .js-focus-visible :focus:not(.${FOCUS_VISIBLE_SELECTOR}):not([data-focus-visible-added]),
+    .js-focus-visible :focus-visible:not(.${FOCUS_VISIBLE_SELECTOR}):not([data-focus-visible-added]) {
```

Comment:
Yeah, this was a bit weird - I think it's a conflict between the S-C library and the `focus-visible` polyfill. The latter adds the `data-focus-visible-added` added property to the new tab when switching using keyboard, but not the correct class. As far as I can see in t[he source of that library](https://github.com/WICG/focus-visible/blob/10fdfa7ad4abc2c7310ecc6daba5f79d08ef90bb/src/focus-visible.js#L83-L84), that shouldn't be possible ever!